src/xcode/ENA/ENA/Source/Views/HairlineConstraint.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors /
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+
+class HairlineConstraint: NSLayoutConstraint {
+	
+	override func awakeFromNib() {
+		super.awakeFromNib()
+		self.constant = 1.0 / UIScreen.main.scale
```

Comment:
What do you think about this:

```suggestion
		constant = 1.0 / ((firstItem as? UIView)?.window?.screen.scale) ?? UIScreen.main.scale
```

Reply:
Thanks for the review @ChristianKienle 

Your code does not compile but I'll apply a similar change.